src/apollo/queries/index.ts

```diff
@@ -66,10 +65,14 @@ export const TOKEN_DERIVED_ETH = (tokenAddress: string) => {
 }
 
 export const GLOBAL_DATA = (chainId: ChainId, block?: number) => {
+  const factoryIdsString = [STATIC_FEE_FACTORY_ADDRESSES[chainId], DYNAMIC_FEE_FACTORY_ADDRESSES[chainId]]
+    .map(address => `"${address ? address.toLowerCase() : ''}"`)
```

Comment:
for arbitrum, velas, oasis, aurora, we should sum legacy contract here because it's still have liquidity. So I suggest removing where condition

Reply:
yeah they will have 2 static fee factories right?, I will remove

Reply:
yes. thanks